etc/docker/hangar-test-maven/include/project-1/test-1.bats

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bats
+
+@test "Compiled target jar exists" {
+  result="$(ls -l ./project-1/target/hangar-test-0.0.1-SNAPSHOT.jar | wc -l)"
+  [ "$result" -eq 1 ]
```

Comment:
Should be:
`[ -e ./project-1/target/hangar-test-0.0.1-SNAPSHOT.jar ]` perhaps? not used `bats` before mind, that's how I would check a file exists in sh